File: GettingStarted/gcdLcm.cpp
#include <iostream>
using namespace std;
    
int LCM(int num1, int num2, int gcd)
{
    return ((num1/gcd)*num2);
}

int GCD(int num1, int num2)
{
    int divisor= num1;
    int dividend= num2;
    while(dividend%divisor!=0){
        int rem=dividend%divisor;
        dividend=divisor;
        divisor=rem;
    }
    return divisor;
}
int main(int argc, char **argv){
    int num1, num2;
    cin>>num1>>num2;
    int gcd=GCD(num1,num2);
    cout<<GCD(num1,num2)<<endl;
    cout<<LCM(num1,num2,gcd);
    return 0;
}
    

File: GettingStarted/gcdLcm.java
import java.util.*;
    
    public class Main{
    public static Scanner scn = new Scanner(System.in);
    public static int LCM(int a, int b, int gcd){
        return ((a/gcd)*b);
    }
    
    public static int GCD(int a, int b){
        int divisor =a;
        int dividend=b;
        while(dividend%divisor!=0){
            int rem= dividend%divisor;
            dividend=divisor;
            divisor=rem;
        }
        return divisor;
    }
    public static void main(String[] args) {
      int a= scn.nextInt();
      int b= scn.nextInt();
      int gcd= GCD(a,b);
      System.out.println(GCD(a,b));
      System.out.println(LCM(a,b,gcd));
     }
    }